src/components/Documentation/Layout/SidebarMenu/styles.module.css

```diff
@@ -60,10 +60,6 @@
 
   &:hover {
     color: #3c3937;
-
-    @media (--xs-scr) {
```

Comment:
@iKBAHT why do we remove this?

Reply:
This fixed bug.

Steps:
- Open docs in mobile
- Open sidebar
- Click to "Install"

Exp: "Install" is highlighted
Act: "Install" remains as it was

Reply:
Not an easy to test in the preview environment for me (your change - this one removal) to see the bug you mention. But this one removal introduces another one for me - when I move the sidebar with my finger it highlights (and keep it highlighted) any item under my finger. So, you kinda see two items highlighted at once.

Reply:
Sorry, it is my fail. I fix this and fix the bug that I found.